todo-app/src/components/CreateTodoList.js

```diff
@@ -29,8 +35,26 @@ const CreateTodoList = () => {
       )}
       {isNameList && (
         <div>
-          <h1>{nameList}</h1>
-          <button onClick={editNameListHandler}>Edit</button>
+          <div>
+            <h1>{nameList}</h1>
+            <button onClick={editNameListHandler}>Edit</button>
+          </div>
+          <form onSubmit={addTodoHandler}>
```

Comment:
I think it would be the best if we go live through this so I can show you some improvements, but you cant start with a few changes I have listed